src/build/java/build/please/compile/FileFinder.java

```diff
@@ -0,0 +1,48 @@
+package build.please.compile;
+
+import java.nio.file.FileVisitResult;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.nio.file.FileVisitResult.CONTINUE;
+
+class FileFinder extends SimpleFileVisitor<Path> {
+
+  private final String extension;
+  private final List<String> files = new ArrayList<>();
```

Comment:
Generally it just gets passed into getJavaFileObjectsFromStrings, so any iterable will do. Might as well change it...